examples/configurable/configurable.description

```diff
@@ -3,12 +3,32 @@
 // variables with values provided externally. The `configurable` keyword can
 // be applied to a module-level variable declaration that contains an initializer.<br/>
 //
-// The values for `configurable` variables can be provided in a file named
-// `Config.toml`. The file location can be set through an environment variable with
-// the name `BALCONFIGFILE`. If an environment variable is not specified, it is located
-// in the current directory.
-//
-// The configuration file must be in the TOML(v0.4) format.
-// Ballerina currently supports configurable variables of types `int`, `float`,
-// `boolean`, `string`, `decimal`, the arrays of the respective types, record, and
-// table.
+// The values for `configurable` variables can be provided through configuration files,
+// CLI parameters, and environment variables.
+// The precedence order for retrieving configurable values is as follows:
+//
+// * CLI parameters with the following format
+// ```
+// -Ckey=value
```

Comment:
Shall we add the key format as well. 

key:= [[orgname .] moduleName .] variable
orgname and module name is optional if the variable defined in the root module.